recipe/meta.yaml

```diff
@@ -24,13 +24,23 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
     - cmake
     - ninja
   host:
     - libqdldl ={{ qdldlversion }}
+    - numpy # For unittests
+    - scipy =1.11.4 # For unittests, as of 0.6.3 imports randn from scipy
 
 test:
+  source_files:
+    - build/out/osqp_demo*
+    - build/out/osqp_tester*
```

Comment:
These files weren't included in the build, and thus haven't been made relocatable. You'd need to point to `$PREFIX/lib/libosqp.so` instead of the one in `$SRC_DIR/build/...`, e.g. using something like patchelf (linux) / install_name_tool (osx).

Or you install those demo files directly to `$PREFIX/...` during installation; the result is essentially the same, because everything under `test.source_files:` gets packaged in the final artefact anyway (to be able to run tests locally)

Reply:
I have installed them in `$PREFIX` as you suggested, I have just used cmake default `install()` behaviour which means they are installed in the `$PREFIX/bin` folder, is there a more suitable place to put them?